### DISCUSSION TITLE
Data folder missing from repo?

### DISCUSSION THREAD

**Discussion by Joebjammin** [TerriaJS/terriajs#6429] [Q&A] [answered]
Hello, After installing a fresh install or TerriaMap, I was unable to find the data folder in wwwroot. It appears to have been deleted from repo. See: https://github.com/TerriaJS/TerriaMap/tree/main/wwwroot

**Comment by nf-s** [comment, marked as answer]
Hi @Joebjammin 

I just replied to your comment 

https://github.com/TerriaJS/TerriaMap/commit/6222c98e15f26e571f939394eab0edb4119b58d1#commitcomment-78732635

> 
> @Joebjammin this was removed as more up-to-date region mapping JSON files are in `terriajs` - https://github.com/TerriaJS/terriajs/tree/main/wwwroot/data
> 
> All the other files in `TerriaMap/wwwroot/data` are out-of-date
> 
> The `data` directory can be re-added if you were hosting files from it
>

**Reply by Joebjammin** [reply]
I was trying to reinstall to apply fix provided in #6397. Can you return to that. Thanks Nick.
